chrisj117/CS-472-GROUP-2-PROJECT
# Review Page Partial Setup

**_WHAT DOES THIS PR DO?_**

  - This PR adds features that will be used in the final Review page (Review.jsx). These include:
     - A review button
     - An additional comments box
     - A professor drop down menu
  - Full functionality, element positioning, and example data are not finalized
  - To access the Review page, a temporary alteration was made to the page routing (main.jsx)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

  - #5 
  - #7 
  - #10
  - #43

**_HOW DO I TEST OUT THIS PR?_**

  - Run this project locally
  - Navigate to the "Review" page by going to http://localhost:5173/review
  - Upon loading, the following page should appear:
![image](https://github.com/chrisj117/CS-472-GROUP-2-PROJECT/assets/35480893/55ff0f52-0a76-4e48-982b-572475dee4b9)
